weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/callsTableQuery.ts

```diff
@@ -120,20 +125,87 @@ export const useCallsForQuery = (
     callsStats.refetch();
   }, [calls, callsStats, costs]);
 
+  // query for annotation feeback
+  const {useFeedbackQuery} = useWFHooks();
+  const feedbackQueryQuery = useMemo(() => {
+    const callRefs = callResults.map(call =>
+      makeRefCall(entity, project, call.callId)
+    );
+    return {
+      $expr: {
+        $and: [
+          {
+            $contains: {
+              input: {$getField: 'feedback_type'},
+              substr: {$literal: `${ANNOTATION_FEEDBACK_TYPE_PREFIX}.`},
```

Comment:
annotation type currently, but what about other feedbacks?

Reply:
we can add to this query, or make the prefix less restrictive